secboot/secboot_tpm.go

```diff
@@ -397,6 +401,51 @@ func ResealKeys(params *ResealKeysParams) error {
 	return sbSealedKeyObjectRevokeOldPCRProtectionPolicies(sealedKeyObjects[0], tpm, authKey)
 }
 
+func TpmPrepare(dir string) error {
```

Comment:
for consistency this needs to be called TPMPrepare, it's a bit of a vogue name though to me, so a more precise name would be better.

Also this bit need a +1 from Chris I think.

This needs a doc comment as well. It would be good if dir had also a more clear name, what kind of dir.

Reply:
> Strike a pose, Vogue!

I guess you mean vague =) but the song reference made my day.

But yes, it is vague, and is opaque as to what sort of things we are attempting to import into tpm, and if they match a given tpm, and it is unknown if they already are known, etc. And yes, this bit of code came from @chrisccoulson already anyway. I'll try to give it more descriptive name and docstring to explain what it does.

Reply:
yes, I meant vague not https://www.youtube.com/watch?v=GuJQSAiODqI :)